docs/recipes/integration-test/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "scripts": {
+    "test": "jest example-lh-auth.test.js"
+  },
+  "dependencies": {
+    "jest": "^24.9.0",
```

Comment:
I also tried to just use `../../../node_modules/.bin/jest`, but we are some versions behind and are missing `this.utils.DIM_COLOR`.